packages/@aws-cdk/cloud-assembly-schema/README.md

```diff
@@ -0,0 +1,62 @@
+## Cloud Assembly Schema
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
+
+
+---
+<!--END STABILITY BANNER-->
+
+This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+
+## Cloud Assembly
+
+The *Cloud Assembly* is the artifact of the AWS CDK class library. It is produced as part of the
```

Comment:
I actually feel your suggestion is less clear. Why do we need to use `construct tree` and `serialization`? This README is not intended solely for developers. We will reference it from the user guide and I think we should avoid being overly technical. 

How about:

```suggestion
The *Cloud Assembly* is the output of the `synthesis` operation.
```

?

Reply:
Sure. what the purpose of the output is or what it contains (so as to infer the former). Otherwise, why is there even an output?

(Also, drop synthesis from backquotes)

Reply:
I mention what this output contains in the following paragraph, is that sufficient?